src/libcollections/btree/node.rs

```diff
@@ -1453,25 +1333,27 @@ unsafe fn move_edges<K, V>(
     dest.correct_childrens_parent_links(dest_offset, dest_offset + count);
 }
 
-impl<BorrowType, K, V, HandleType>
-        Handle<NodeRef<BorrowType, K, V, marker::LeafOrInternal>, HandleType> {
-
+impl<BorrowType, K, V, HandleType> Handle<NodeRef<BorrowType, K, V, marker::LeafOrInternal>,
+                                          HandleType> {
```

Comment:
I think it makes more sense to split the `impl` and the type than it does to split the type in half.